src/PBot.Tests.Integration/List_my_TaskForces.cs

```diff
@@ -0,0 +1,131 @@
+﻿namespace PBot.Tests.Integration
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using Octokit;
+
+    public class List_my_TaskForces //: BotCommandFixture<ListIssuesFromExternalContributors>
+    {
+        [Test, Explicit]
+        public async void All()
+        {
+            Console.Out.WriteLine( "### My Task forces" );
```

Comment:
why console write? does this get redirected to some sensible place when run in the context of the bot?